contrib/babelfishpg_tsql/src/guc.c

```diff
@@ -605,6 +606,16 @@ define_custom_variables(void)
 				 GUC_NO_SHOW_ALL,
 				 NULL, NULL, NULL);
 
+	/* temporary GUC for enable or disable windows login */
+	DefineCustomBoolVariable("babelfishpg_tsql.pltsql_is_windows_allowed",
+				 gettext_noop("GUC for enable or disable windows login"),
+				 NULL,
+				 &pltsql_is_windows_allowed,
```

Comment:
pltsql is not needed in GUC name as "babelfishpg_tsql" is already there. "is_windows_allowed" does not convey the whole intent, if it is windows login, login should be part of guc name like 'allow_windows_login"

Reply:
Done!